client/components/disputed-order-notice/index.js

```diff
@@ -0,0 +1,210 @@
+		),
+		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
+		inquiry_default: __(
+			// eslint-disable-next-line max-len
+			'The card network involved in this order has opened an inquiry into the transaction with the following reason: %2$s. Please respond to this inquiry before %3$s, just like you would for a formal dispute.',
+			'woocommerce-payments'
+		),
+		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
+		dispute_urgent: __(
+			'Please resolve the dispute on this order for %1$s labeled "%2$s" by %3$s.',
+			'woocommerce-payments'
+		),
+		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
+		inquiry_urgent: __(
+			'Please resolve the inquiry on this order for %1$s labeled "%2$s" by %3$s.',
+			'woocommerce-payments'
+		),
+	};
+
+	let buttonLabel = __( 'Respond now', 'woocommerce-payments' );
+	let suffix = '';
+
+	let titleText = isPreDisputeInquiry
+		? titleStrings.inquiry_default
+		: titleStrings.dispute_default;
+
+	// If the dispute is due within 7 days, adjust wording and highlight urgency.
+	if ( countdownDays < 7 ) {
+		titleText = isPreDisputeInquiry
+			? titleStrings.inquiry_urgent
+			: titleStrings.dispute_urgent;
+
+		suffix = sprintf(
+			// Translators: %s is the number of days left to respond to the dispute.
+			_n(
+				'(%s day left)',
+				'(%s days left)',
+				countdownDays,
+				'woocommerce-payments'
+			),
+			countdownDays
+		);
+	}
+
+	const title = sprintf(
+		titleText,
+		formattedAmount,
+		reasons[ disputeReason ].display,
+		dateI18n( 'M j, Y', dueBy.local().toISOString() )
+	);
+
+	if ( countdownDays < 1 ) {
+		buttonLabel = __( 'Respond today', 'woocommerce-payments' );
+		suffix = __( '(Last day today)', 'woocommerce-payments' );
+	}
+
+	return (
+		<InlineNotice
+			status={ countdownDays < 3 ? 'error' : 'warning' }
+			isDismissible={ false }
+			actions={ [
+				{
+					label: buttonLabel,
+					variant: 'secondary',
+					onClick: () => {
+						wcpayTracks.recordEvent(
+							'wcpay_order_dispute_notice_action_click',
+							{
+								due_by_days: countdownDays,
+							}
+						);
+						window.location = disputeDetailsUrl;
+					},
+				},
+			] }
+		>
+			{ <strong>{ `${ title } ${ suffix }` }</strong> }
+		</InlineNotice>
+	);
+};
+
+const DisputeUnderReviewNotice = ( { disputeDetailsUrl } ) => {
+	return (
+		<InlineNotice status="warning" isDismissible={ false }>
+			{ createInterpolateElement(
+				__(
+					'This order has an active payment dispute. ' +
+						'Refunds and order editing are disabled. <a>View details</a>',
+					'woocommerce-payments'
+				),
+				{
+					// createInterpolateElement is incompatible with this eslint rule as the <a> is decoupled from content.
+					// eslint-disable-next-line jsx-a11y/anchor-has-content
+					a: <a href={ disputeDetailsUrl } />,
+				}
+			) }
+		</InlineNotice>
+	);
+};
```

Comment:
The joy of a simple component that just does what it says, and can be easily used whereever!

Reply:
~~The joy of a simple component that just does what it says, and can be easily used whereever!~~

Is now slightly more complex … but I'm ok with that as the purpose is clear: explain to the merchant why the order is not editable.